save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/basic/FileUploader.kt

```diff
@@ -81,9 +81,10 @@ fun fileUploader(
                 }
                 li("list-group-item d-flex justify-content-between align-items-center") {
                     select(classes = "form-control") {
+                        attrs.value = "default"
```

Comment:
Why do we need this twice?

Reply:
It's for different tags - for `select` and `option`. React encourages to use `value` on `select` instead of `selected` on `option`.